editor/src/clj/editor/model_loader.clj

```diff
@@ -60,6 +64,7 @@
       {:mesh-set mesh-set
        :skeleton skeleton
        :bones bones
+       :buffers (.buffers scene)
```

Comment:
Structs of `{ String uri; byte[] buffer; }` that allows the editor to add proper resource dependencies.